Week7/week7.py

```diff
@@ -0,0 +1,66 @@
+import csv
+with open ("E://sem-3//product.csv",'w') as f:
+    data=csv.writer(f)
+    header=["prod_no","prod_name","jan","feb","mar","apr","may","jun"]
+    data.writerow(header)
+    for i in range(12):
+        prod_no=int(input("Enter product no:"))
+        prod_name=input("Enter name:")
+        jan=int(input("Enter product selling of jan:"))
+        feb=int(input("Enter product selling of feb:"))
+        mar=int(input("Enter product selling of mar:"))
+        apr=int(input("Enter product selling of apr:"))
+        may=int(input("Enter product selling of may:"))
+        jun=int(input("Enter product selling of jun:"))
+        record=[prod_no,prod_name,jan,feb,mar,apr,may,jun]
+        data.writerow(record)
+        
+with open("E://sem-3//product.csv",'r')as f1:
+    content=csv.reader(f1)
+    for i in content:
+        print(i)
+#2.Create dataframe.        
+import pandas as pd
+df=pd.read_csv("E:\sem-3\product.csv")

```

Comment:
Explain this statement?
when you read csv file then use forward slash '//' and then you create dataframe you use backward slash '\'